hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -242,3 +266,11 @@ def collect_rendered_images(self, parent_item):
                 # the item has been created. update the display name to include
                 # the an indication of what it is and why it was collected
                 item.name = "%s (Render Layer: %s)" % (item.name, layer)
+
+    def _alembic_cache_dir(self, project_root):
```

Comment:
This seems unnecessary since `collect_alembic_caches` checks for the path's existence first thing.